### PR TITLE
PLAT-798 | Create orders list in admin

### PR DESCRIPTION
**Pull request by kvas-damian**
After the release run `php artisan scout:import App\\Models\\Order`

### CODE REVIEW

File: resources/assets/js/admin/components/orders/OrdersList.vue [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+			</thead>
+			<tbody>
+			<tr v-for="order in orders" @click="goToOrder(order)" class="row">
+				<td>{{order.id}}</td>
+				<td>{{order.created_at}}</td>
+				<td>{{order.user_id}}</td>
+				<td>{{order.product.name}}</td>
+				<td>{{translateShippingStatus(order)}}</td>
+				<td>
+					<span class="icon has-text-success	" v-if="order.paid"><i class="fa fa-check"></i></span>
```
Comment (bognix) [inline resources/assets/js/admin/components/orders/OrdersList.vue]: dziwne spacje 🙈

File: resources/assets/js/admin/components/orders/OrdersList.vue [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+				<th>Data</th>
+				<th>Id użytkownika</th>
+				<th>Produkt</th>
+				<th>Status wysyłki</th>
+				<th>Wpłata</th>
+				<th>Kupon</th>
+			</tr>
+			</thead>
+			<tbody>
+			<tr v-for="order in orders" @click="goToOrder(order)" class="row">
```
Comment (bognix) [inline resources/assets/js/admin/components/orders/OrdersList.vue]: v-for with :key pretty please

File: resources/assets/js/admin/components/orders/OrdersList.vue [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+				this.fetch();
+			},
+			async onSearch({phrase}) {
+				this.page = 1;
+				this.searchPhrase = phrase;
+
+				await this.fetch()
+			},
+			async fetch() {
+				const response = await axios.post(getApiUrl('orders/.filter'), this.getRequestParams())
```
Comment (bognix) [inline resources/assets/js/admin/components/orders/OrdersList.vue]: not related with your changes but automatically including coupons, studyBuddies and product data for each order (I'm talking about Transformer) costs us unnecessary DB calls. Maybe you would like to move one include out of transformer :)

File: resources/assets/js/admin/components/users/UserDetails.vue [hunk 4]
```diff
@@ -160,10 +160,16 @@
 			changeTab(name) {
 				this.activeTab.active = false;
 				this.tabs[name].active = true;
+				this.$router.replace({ hash: `#${name}` })
```
Comment (bognix) [inline resources/assets/js/admin/components/users/UserDetails.vue]: can't this be done using <router-link>. This way you could avoid removing and adding the hash in mounted.

Comment (kvas-damian) [reply resources/assets/js/admin/components/users/UserDetails.vue]: <div>asd</div>

Comment (bognix) [reply resources/assets/js/admin/components/users/UserDetails.vue]: <script>alert('to ja')</script>

Comment (kvas-damian) [reply resources/assets/js/admin/components/users/UserDetails.vue]: <div onclick="alert(1)">qwert</div>

**Pull request merged**